### DISCUSSION TITLE
Problems with not being able to search only specific URLs

### DISCUSSION THREAD

**Discussion by other-other-yuka** [lavalink-devs/Lavalink#840] [Q&A] [answered]
## Description

when I searched by title, it played normally, but when I searched by URL, it didn't play well.

```
ytsearch:https://www.youtube.com/watch?v=_5Yh5a5AM18
```

```
ytsearch:[PLAYLIST] 누나가 말했지 나보고 힙찔이라고 | 힙합 playlist
```

## Version info
**Client used:** 

Output of `java -version`:
```
openjdk version "18.0.2.1" 2022-08-18
OpenJDK Runtime Environment Temurin-18.0.2.1+1 (build 18.0.2.1+1)
OpenJDK 64-Bit Server VM Temurin-18.0.2.1+1 (build 18.0.2.1+1, mixed mode, sharing)
```

```
lavalink  | 2023-02-24 17:17:47.482  INFO 1 --- [  XNIO-1 task-2] l.server.player.AudioLoaderRestHandler   : Got request to load for identifier "ytsearch:https://www.youtube.com/watch?v=_5Yh5a5AM18"
lavalink  | 2023-02-24 17:17:47.683  INFO 1 --- [ader-2-thread-1] lavalink.server.player.AudioLoader       : **No matches found**

lavalink  | 2023-02-24 17:17:55.104  INFO 1 --- [  XNIO-1 task-1] l.server.player.AudioLoaderRestHandler   : Got request to load for identifier "ytsearch:[PLAYLIST] 누나가 말했지 나보고 힙찔이라고 | 힙합 playlist"
lavalink  | 2023-02-24 17:17:55.353  INFO 1 --- [ader-2-thread-1] lavalink.server.player.AudioLoader       : **Loaded playlist Search results for**: [PLAYLIST] 누나가 말했지 나보고 힙찔이라고 | 힙합 playlist
lavalink  | 2023-02-24 17:17:55.359  INFO 1 --- [   XNIO-1 I/O-1] lavalink.server.io.SocketServer          : {"op":"play","guildId":"766306149754077194","track":"QAAAqgIASltQTEFZTElTVF0g64iE64KY6rCAIOunkO2WiOyngCDrgpjrs7Tqs6Ag7Z6Z7LCU7J2065286rOgIHwg7Z6Z7ZWpIHBsYXlsaXN0AAbrrLTrhIgAAAAAAEH6oAALXzVZaDVhNUFNMTgAAQAraHR0cHM6Ly93d3cueW91dHViZS5jb20vd2F0Y2g/dj1fNVloNWE1QU0xOAAHeW91dHViZQAAAAAAAAAA","noReplace":true,"pause":false}
```

```
lavalink  |   Version:        3.7.4
lavalink  |   Build time:     02.02.2023 18:34:07 UTC
lavalink  |   Branch          HEAD
lavalink  |   Commit:         8ebaba1
lavalink  |   Commit time:    02.02.2023 18:32:05 UTC
lavalink  |   JVM:            18.0.2.1
lavalink  |   Lavaplayer      1.3.99.2-original
```

**Comment by topi314** [comment, marked as answer]
Links already point to a specific video
So why would you need to search them

**Reply by topi314** [reply]
Searching URLs is not supported

(You reply under an answer
Not create new answers)

**Reply by other-other-yuka** [reply]
lavalink  | 2023-02-24 17:38:26.896  INFO 1 --- [  XNIO-1 task-2] l.server.player.AudioLoaderRestHandler   : Got request to load for identifier "ytsearch:https://youtu.be/qohUYCQlM94"
lavalink  | 2023-02-24 17:38:27.256  INFO 1 --- [ader-2-thread-1] lavalink.server.player.AudioLoader       : Loaded playlist Search results for: https://youtu.be/qohUYCQlM94
lavalink  | 2023-02-24 17:38:27.267  INFO 1 --- [  XNIO-1 task-2] l.server.player.AudioLoaderRestHandler   : Got request to load for identifier "ytsearch:[Playlist] 틀어놓으면 첫곡부터 극락가는 K-POP 노래모음 플레이리스트 | 케이팝 노동요 & 매장음악 플레이리스트"
lavalink  | 2023-02-24 17:38:27.788  INFO 1 --- [ader-2-thread-1] lavalink.server.player.AudioLoader       : Loaded playlist Search results for: [Playlist] 틀어놓으면 첫곡부터 극락가는 K-POP 노래모음 플레이리스트 | 케이팝 노동요 & 매장음악 플레이리스트
lavalink  | 2023-02-24 17:38:27.793  INFO 1 --- [   XNIO-1 I/O-1] lavalink.server.io.SocketServer          : {"op":"play","guildId":"766306149754077194","track":"QAAA8gIAkltQbGF5bGlzdF0g7YuA7Ja064aT7Jy866m0IOyyq+qzoeu2gO2EsCDqt7nrnb3qsIDripQgSy1QT1Ag64W4656Y66qo7J2MIO2UjOugiOydtOumrOyKpO2KuCB8IOy8gOydtO2MnSDrhbjrj5nsmpQgJiDrp6TsnqXsnYzslYUg7ZSM66CI7J2066as7Iqk7Yq4AAbtlZHtlIQAAAAAAIxGKAALcW9oVVlDUWxNOTQAAQAraHR0cHM6Ly93d3cueW91dHViZS5jb20vd2F0Y2g/dj1xb2hVWUNRbE05NAAHeW91dHViZQAAAAAAAAAA","noReplace":true,"pause":false}

Why does this work if you don't apply?

**Reply by topi314** [reply]
You're not supposed to search URLs and that's it
It may work because yt
It may not

**Reply by other-other-yuka** [reply]
All the URLs I searched are yt. so I don't know why.

**Reply by topi314** [reply]
You don't put `ytsearch:` Infront of URLs.

That's it

**Reply by other-other-yuka** [reply]
OK.
